packs/data/bestiary-effects.db/effect-xiuh-couatl-venom-stage-2.json

```diff
@@ -0,0 +1,45 @@
+{
+    "_id": "p5jMLEwviw7IdlGf",
+    "data": {
+        "badge": null,
+        "description": {
+            "value": "<p>The creature takes a -5-foot status penalty to all Speeds.</p>"
+        },
+        "duration": {
+            "expiry": "turn-end",
+            "sustained": false,
+            "unit": "rounds",
+            "value": 1
+        },
+        "level": {
+            "value": 4
+        },
+        "rules": [
+            {
+                "key": "FlatModifier",
+                "selector": "speed",
+                "type": "status",
+                "value": -5
+            }
+        ],
+        "source": {
+            "value": "Pathfinder Bestiary 3"
+        },
+        "start": {
+            "initiative": null,
+            "value": 0
+        },
+        "target": null,
+        "tokenIcon": {
+            "show": true
+        },
+        "traits": {
+            "custom": "",
+            "rarity": "common",
+            "value": []
+        }
+    },
+    "img": "systems/pf2e/icons/equipment/alchemical-items/alchemical-poisons/wyvern-poison.webp",
+    "name": "Effect: Xiuh Couatl Venom (Stage 2)",
```

Comment:
Let's wait until we have afflictions.

Reply:
Sure, fine for me - removed the effects.